demos/checkbox.scss

```diff
@@ -15,6 +15,36 @@
 //
 
 @import "./common";
+@import "../packages/mdc-button/mdc-button";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
+@import "../packages/mdc-theme/color_palette";
+@import "../packages/mdc-theme/mixins";
 @import "../packages/mdc-typography/mdc-typography";
+
+.mdc-checkbox.demo-checkbox--custom-all {
+  $color: $material-color-red-500;
+
+  @include mdc-checkbox-focus-indicator-color($color);
+  @include mdc-checkbox-ripple((base-color: $color, opacity: $mdc-checkbox-ripple-opacity));
+  @include mdc-checkbox-container-colors(
+    $unmarked-stroke-color: $color,
+    $unmarked-fill-color: rgba($color, .25),
+    $marked-fill-color: $color);
+
+  @include mdc-theme-dark(".mdc-checkbox", $compound: true) {
+    @include mdc-checkbox-ink-color(black);
+    @include mdc-checkbox-container-colors(
+      $unmarked-stroke-color: $color,
+      $unmarked-fill-color: rgba($color, .35),
+      $marked-fill-color: $color);
+  }
+}
+
+.mdc-checkbox.demo-checkbox--custom-stroke-and-fill {
+  $color: $material-color-blue-500;
+
+  @include mdc-checkbox-container-colors(
+    $unmarked-stroke-color: $color,
```

Comment:
I would suggest making this color different to better illustrate how this mixin's parameters work now. I was seriously confused for several minutes thinking we were applying the same stroke color to both states, but IIUC that's not what we're doing; we're making the marked stroke color match the marked fill color, which makes sense.

Reply:
Done